backend-application/src/main/resources/application.yml

```diff
@@ -35,6 +35,32 @@ ext:
 ---
 
 spring:
+  profiles: demo
```

Comment:
Consider using spring.profiles.include and derive this profile from dev.
https://stackoverflow.com/questions/25985752/including-profiles-with-spring-profiles-include-seems-to-override-instead-of-inc